### DISCUSSION TITLE
Startup Project

### DISCUSSION THREAD

**Discussion by tepdock** [NoiseStudio/NoiseEngine#365] [Q&A] [answered]
@Vixenka 
Hi again! I wanted to ask how to actually start project? I thought I need to set NoiseEngine.Builder as startup project, but i'm getting exception about shaders: 
![image](https://github.com/NoiseStudio/NoiseEngine/assets/70803263/5374b0cc-dcb4-44d5-8daa-05440cd25945)
Is there anything I need to do before I start a project, or have I chosen the wrong project to start?

**Comment by kkard2** [comment, marked as answer]
what do you mean by "starting the project"? currently it's only a "library", so you cannot execute anything (besides tests)
building should work fine, if that's not working then it's not good

**Reply by tepdock** [reply]
Found two Program.cs files, thought you can execute program. Sorry for confusion, thank you for fast answer!
